build.gradle

```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+    ext.kotlin_version = '1.5.31'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.android.tools.build:gradle:4.2.2'
```

Comment:
I think the latest version of the gradle build tools is actually 7.0.3, do you see that in your version of Android Studio?

Reply:
Thanks for this catch! I see it and updated to 7.0.3 in a follow up commit in this pull request.